open-metadata-implementation/common-services/ffdc-services/src/main/java/org/odpi/openmetadata/commonservices/ffdc/rest/Parametrization.java

```diff
@@ -1,27 +1,32 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /* Copyright Contributors to the ODPi Egeria project. */
-package org.odpi.openmetadata.accessservices.subjectarea.client;
+package org.odpi.openmetadata.commonservices.ffdc.rest;
 
-import org.odpi.openmetadata.accessservices.subjectarea.responses.SubjectAreaOMASAPIResponse;
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.core.ResolvableType;
 
 /**
- * @param <T> The type of the parameterized class whose objects will be returned in responses.
+ * @param <Result> The type of the parameterized class whose objects will be returned in responses.
  * The purpose of this interface is to enable capturing and passing a generic type. */
```

Comment:
should "enable capturing and passing a generic type" be "enable capturing and passing a generic result"

Reply:
Okay